configure.ac

```diff
@@ -328,6 +328,21 @@ AC_DEFINE_UNQUOTED([PCMK__CONCURRENT_FENCING_DEFAULT],
                    ["$with_concurrent_fencing_default"],
                    [Default value for concurrent-fencing cluster option])
 
+AC_ARG_WITH([sbd-sync-default],
+    [AS_HELP_STRING([--with-sbd-sync-default], m4_normalize([
+        default value used by sbd if SBD_SYNC_RESOURCE_STARTUP
+        environment variable is not set @<:@false@:>@]))],
+)
+AS_CASE([$with_sbd_sync_default],
+        [""], [with_sbd_sync_default=false],
+        [false], [],
```

Comment:
to denote that PCMK__SBD_SYNC_DEFAULT goes into an 'int' and not a 'bool' we could use 0/1 here.